doc/content/source/materials/NEMLStress.md

```diff
@@ -8,20 +8,27 @@ This material model provides an interface to the constitutive models for structu
 
 NEML permits the definition of complex material constitutive models, the details of which are defined in an XML file that is independent of the MOOSE input file. The required inputs for this model on the MOOSE side simply provide the location of the XML file, which contains a database of potentially multiple material models, and the name of the model within that database to be used.  This model computes the stress, elasticity tensor, and material Jacobian.
 
-Using this model requires that the NEML source code is present, and that the prerequisite libraries are available. NEML is provided as a submodule in BlackBear under the directory `blackbear/contrib/neml`. The up-to-date version of the NEML submodule is obtained using:
+Using this model requires that the NEML source code is present, and that the prerequisite libraries are available. 
+NEML is provided as a submodule in BlackBear under the directory `blackbear/contrib/neml`. 
+The up-to-date version of the NEML submodule is obtained using:
+
 ```
+cd blackbear/contrib/neml
 git submodule init
 git submodule update
 ```
-in the `blackbear` directory.
 
-NEML is compiled using the MOOSE build system, so no separate compilation step is required to build the library. However, NEML has dependencies on the BOOST library that is not present in the default MOOSE environment.
+NEML is compiled using the MOOSE build system, so no separate compilation step is required to build the library. 
+However, NEML has dependencies on the BOOST library that are not present in the default MOOSE environment.
+To build with boost, one must simply load the `boost` module provided with MOOSE:
 
-To build with BOOST, one must simply load the `boost` module provided with MOOSE:
 ```
 module load advanced_modules boost
 ```
-and add the `--with-boost` option to the `update_and_rebuild_libmesh.sh` script when compiling libMesh to instruct it to use the full BOOST instalation provided by the `boost` module, rather than the subset of BOOST that is distributed with libmesh.
+
+and modify the `update_and_rebuild_libmesh.sh` script by adding `--with-boost` to the INSTALL options and rebuilding libMesh.  
```

Comment:
What are the INSTALL options?

Reply:
I think I'd say:

and use the `--with-boost` option while rebuilding libMesh:
```
update_and_rebuild_libmesh.sh --with-boost

Reply:
I didn't know you could pass it in as an option to the libmesh script

Reply:
i'm trying it to see if it builds.

Reply:
Yeah, that's what I tried to say in the original version of this, but clearly it didn't come across right if you didn't catch that! That's why it's good to have someone else try to follow your instructions.

Reply:
By the way, if you didn't already catch this, you can do that in parallel:
```
JOBS=8 ./scripts/update_and_rebuild_libmesh.sh --with-boost